cime_config/config_component.xml

```diff
@@ -24,8 +24,7 @@
     <valid_values>prognostic,prescribed,thermo_only</valid_values>
     <default_value>prognostic</default_value>
     <values>
-      <value compset="CICE_">prognostic</value>
```

Comment:
I deleted this line: It's unnecessary since the default value is already 'prognostic', and it's hard to write a correct regular expression to express the setting of 'prognostic' now that %CMIP6 is a supported modifier.